src/inscription.rs

```diff
@@ -38,9 +40,14 @@ pub struct Inscription {
 
 impl Inscription {
   #[cfg(test)]
-  pub(crate) fn new(content_type: Option<Vec<u8>>, body: Option<Vec<u8>>) -> Self {
```

Comment:
Let's leave this functions is, and create inscriptions which need content encoding with the `Inscription {…}` constructor.